src/docfx/restore/RestoreUrl.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.Docs.Build
 {
     internal static class RestoreUrl
     {
-        private const int MaxVersionCount = 5;
+        public const int MaxKeepingDays = 10;
```

Comment:
shall we keep this as private if this is only used for testing? I mean we can even set the `LastAccessTime` to 20 days ago.
Same for `RestoreWorkTree`

Reply:
ok, sounds good

---
In reply to: [230249986](https://github.com/dotnet/docfx/pull/3564#discussion_r230249986) [](ancestors = 230249986)